README.md

```diff
@@ -2,12 +2,14 @@
 
 [![Build Status](https://travis-ci.org/Infineon/TLE5012-Magnetic-Angle-Sensor.svg?branch=master)](https://travis-ci.org/Infineon/TLE5012-Magnetic-Angle-Sensor)
 
-<img src="https://www.infineon.com/export/sites/default/media/products/Sensors/TLE_5012B_DSO-8-16_plain_lowres.jpg_945841647.jpg" width=200>
+| <img src="https://www.infineon.com/export/sites/default/media/products/Sensors/TLE_5012B_DSO-8-16_plain_lowres.jpg_945841647.jpg" width=200 style="background:white"> | <img src="https://raw.githubusercontent.com/infineon/assets/master/Pictures/TLE5012B_horizontal.png" width=400 style="background:white"> |
+|:--:|:--:|
+| *bulk sensor* | *Sensor2Go kit* 
 
-Library of Infineon's highly sensitive [TLE5012B] 360° magnetic angle sensor(https://www.infineon.com/cms/en/product/sensor/magnetic-position-sensor/angle-sensor/tle5012b-e1000/) for Arduino.
+Library of Infineon's highly sensitive [TLE5012B](https://www.infineon.com/cms/en/product/sensor/magnetic-position-sensor/angle-sensor/tle5012b-e1000/) 360° magnetic angle sensor for Arduino.
```

Comment:
change url to
https://www.infineon.com/cms/en/product/sensor/magnetic-sensors/magnetic-position-sensors/angle-sensors/